react/Image.tsx

```diff
@@ -76,6 +127,19 @@ function Image(props: ImageProps) {
   ) : (
     <Fragment>{imgElement}</Fragment>
   )
+
+  return experimentalPreventLayoutShift ? (
+    <span
```

Comment:
Why `span` and not a `div` or something else?

Reply:
@Klynger I think `<span>` is more akin to how images work (i.e. predominantly inline, rather than a block) but same difference I guess